EveripediaNetwork/ep-ui
# Feat/hiiq big number

# Fixing hiiq number wrong representation [#1201](https://github.com/EveripediaNetwork/issues/issues/1201)

_On the wallet details card, the hiiq number is not well represented. The token amount is very large and needs to be properly formatted like a crypto token and not in exponential._

## How should this be tested?

1. Click the wallet icon
2. Check the hiiq balance to see if it is well formatted.


## Linked issues

https://github.com/EveripediaNetwork/issues/issues/1201
